### DISCUSSION TITLE
AutoDM has stopped working

### DISCUSSION THREAD

**Discussion by Hellifix** [marioparaschiv/nitro-sniper#385] [Q&A]
It used to be OK, but recently AutoDM has stopped working for me. After winning, autodm could not write to DM, and in the log it wrote that it could not write to the person. This pissed me off so I started looking for a solution to this issue, initially I created two clean accounts with DM enabled, and privacy settings, also created a separate server with Giveaway bot. On the first account, I did test giweaways, and the second was waiting for an answer in DM from the bot, but the bot kept failing to write, I also tried to change some parameters relating to the DM in setting sniper, as well as putting the default settings, but it did not help( Please help solve this issue, for the past week, trying to solve it (

![20220427_154612](https://user-images.githubusercontent.com/101359230/165511532-57ac70e3-aa9a-4f3c-bc15-a53596ce13b2.jpg)

**Comment by Craftzman7** [comment]
Check the receiver's and sender's privacy settings.

**Reply by Hellifix** [reply]
The privacy settings were still set to normal. I also wanted to note that when the sender's account is already in my friends, or has already started a dialog with that account, AutoDM works, I don't know how it works, I think I'm one of the first people to catch this bug with sniper bot

**Reply by Craftzman7** [reply]
This is not a bug. I tested it.

**Reply by Hellifix** [reply]
How could this problem be fixed?
I don't even know what caused it, I don't think I touched anything with the settings etc. It just started on its own

**Reply by Burri61** [reply]
i have the same problem

**Reply by Craftzman7** [reply]
> i have the same problem
> 
> 

Privacy settings issue.
